src/services/session.service.ts

```diff
@@ -0,0 +1,32 @@
+import Session from "../session/session";
```

Comment:
This file should be covered by unit tests as well

Reply:
Unit test added.